site/src/components/getting-started/overview/OverviewCards.tsx

```diff
@@ -15,6 +16,18 @@ export const OverviewCard = ({
   />
 );
 
-export const OverviewCards = ({ children }: { children: ReactNode }) => (
-  <div className={styles.overviewCards}>{children}</div>
+export const OverviewCards = ({
+  children,
+  columns = 2,
+}: {
+  children: ReactNode;
+  columns?: 2 | 3;
+}) => (
+  <div
```

Comment:
Can one of our Layout components be used for this instead?

Reply:
Probably. :-)

Will have a look into that tomorrow